Lib/test/test_dis.py

```diff
@@ -1050,12 +1105,12 @@ def test_explicit_first_line(self):
         self.assertEqual(list(actual), expected_opinfo_outer)
 
     def test_source_line_in_disassembly(self):
-        # Use the line in the source code
-        actual = dis.Bytecode(simple).dis()[:3]
+        # Use the line in the source code (split extracts the line no)
+        actual = dis.Bytecode(simple).dis().split(" ")[0]
```

Comment:
`expected` hasn't been defined yet, so we'd have to change the order (raising the point that we would make an order-independent setup order-dependent). It also breaks consistency with the second comparison; closest w/o refactoring would to be `[:len("350")]`, repeating the magic number a third time, just as `[:3]` encodes the length of the magic number.

If it were up to me I'd just pick `.partition(" ")[0]`; in addition to the control flow simplicity and consistency I think it's clearer (we're splitting the string up and picking part of it). @serhiy-storchaka , do you feel really strongly about this? @ncoghlan , do you prefer `[:len(expected)]` (w/ `[:len('350')]` ) as well?